### PR TITLE
Auto version pulling

### PR DESCRIPTION
**Pull request by misxki**
Just automatic version pulling from Github releases/latest for the spacedrive landing page (so it doesn't have to be manually).

### CODE REVIEW

File: apps/landing/src/pages/index.tsx [hunk 1]
```diff
@@ -124,6 +124,10 @@ export default function HomePage() {
 		};
 	}, []);
 
+	useEffect(() => {
+		getLatestSpacedriveVersion().then((res) => setSpacedriveVersion(res));
```
Comment (PineappleRind) [inline apps/landing/src/pages/index.tsx:128]:
```suggestion
		getLatestSpacedriveVersion().then(setSpacedriveVersion);
```
shorter

**Comment by Brendonovich** [comment]
Once #1587 is merged I'm going to implement changelog pulling on the server, it'd use a similar fetch as you're doing but on the server instead. Would you like to re-implement this to fetch server-side in a different PR once I'm done with the changelog stuff?

**Comment by misxki** [comment]
Yea sure, I can do that!

**Comment by Brendonovich** [comment]
#1592 is done now, feel free to implement this! Don't use any client components though.

**Pull request closed without merging**